rts/Lua/LuaOpenGL.cpp

```diff
@@ -4625,37 +4627,78 @@ int LuaOpenGL::GetWaterRendering(lua_State* L)
 
 int LuaOpenGL::GetMapRendering(lua_State* L)
 {
-	const char* key = luaL_checkstring(L, 1);
-
-	switch (hashString(key)) {
-		// float4
-		case hashString("splatTexScales"): {
```

Comment:
why was the switch/case replaced?

Reply:
It clearly trolled me when rebasing... I'm restoring case later!